plugins/out_kafka/CMakeLists.txt

```diff
@@ -14,4 +14,8 @@ set(src
   kafka.c)
 
 FLB_PLUGIN(out_kafka "${src}" "rdkafka")
-target_link_libraries(flb-plugin-out_kafka -lpthread)
+if (MSVC)
+  target_link_libraries(flb-plugin-out_kafka)
```

Comment:
note that line is doing nothing, the condition must be "if i am not on MSVC, link to pthread"

Reply:
Oops, that's too bad.

Reply:
I've fixed at https://github.com/fluent/fluent-bit/pull/1179/commits/f95ad547aa619585270029602163fca564f77372.